### DISCUSSION TITLE
复制av号链接

### DISCUSSION THREAD

**Discussion by yangyangbuxiangyang** [the1812/Bilibili-Evolved#3794] [提问 / Q&A] [answered]
av号和bv号后面的复制链接按钮复制出来的内容都是bv号的链接，是原本就这么设计的还是bug？

**Comment by Geeyun-JY3** [comment]
显然是 bug。
而且这 bug 应该说是复制出来的都是当前网页地址栏显示的 URL。

**Comment by Ethkuil** [comment]
我打算尝试fix这个bug，就当做练习。刚搭好开发环境并完成复现。

另外还发现一个相关bug，复制链接时带上的标题是重复过一遍的结果。

**Comment by Ethkuil** [comment, marked as answer]
两个都修好了。我发个PR。

**Reply by Ethkuil** [reply]
#3797
